src/components/Fuel/FuelForm.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from "react";
```

Comment:
Can you run Prettier on this page? It looks wonky and not consistent with the others.

Reply:
Done!